setup.py

```diff
@@ -35,6 +35,7 @@
 
 try:
     import sphinx
+    import sphinx.cmd.build
```

Comment:
sphinx.cmd has only existed since late 2017, so this import is likely to fail on versions of Sphinx people get from Linux distro package repos. You'll have to try to import it in its own try block instead.

The commit that create sphinx.cmd is here: https://github.com/sphinx-doc/sphinx/commit/89f9c7cab74298a243ac409c26d25eb2b137bf03#diff-6fa83f2439b109c5dc63ceb41b3e51f0